__tests__/app.test.js

```diff
@@ -184,25 +183,85 @@ describe("POST /api/articles/:article_id/comments", () => {
-    .then((response) => {
-      expect(response.body.msg).toBe("this article id does not exist yet");
-    });
+  test("404:responds with an error, this article_id does not exist yet", () => {
+    return request(app)
+      .post("/api/articles/100/comments")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("this article id does not exist yet");
+      });
+  });
+  test("404:responds with an error, this username does not exist yet", () => {
+    const newComment = {
+      username: "RonBurgundy",
+      body: "60% of the time it works every time",
+    };
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(newComment)
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe(
+          "Sorry, but this username doesn't exist."
+        );
+      });
+  });
 });
-test("404:responds with an error, this username does not exist yet", () => {
-  const newComment = {
-    username: "RonBurgundy",
-    body: "60% of the time it works every time",
-  };
-  return request(app)
-    .post("/api/articles/2/comments")
-    .send(newComment)
-    .expect(404)
-    .then((response) => {
-      expect(response.body.msg).toBe("Sorry, but this username doesn't exist.");
-    });
+
+describe("PATCH REQUEST/api/articles/:article_id", () => {
+  test("200: update the votes in articles (+)", () => {
+    const newVote = { inc_votes: 1 };
+    const expectedArticle1 = {
+        article_id: 1,
+        title: 'Living in the shadow of a great man',
+        topic: 'mitch',
+        author: 'butter_bridge',
+        body: 'I find this existence challenging',
+        created_at: '2020-07-09T20:11:00.000Z',
+        votes: 101,
+        article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+      }
+    return request(app)
+      .patch("/api/articles/1")
+      .send(newVote)
+      .expect(200)
+      .then((response) => {
+        console.log(response.body);
+        expect(response.body).toMatchObject(expectedArticle1);
+        expect(response.body.votes).toBe(101);
+      });
+  });
+  test("200: update the votes in articles (-)", () => {
+    const newVote = { inc_votes: -1 };
+    const expectedArticle1 = {
+      article_id: 1,
+      title: 'Living in the shadow of a great man',
+      topic: 'mitch',
+      author: 'butter_bridge',
+      body: 'I find this existence challenging',
+      created_at: '2020-07-09T20:11:00.000Z',
+      votes: 100,
+      article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700'
+    }
+    return request(app)
+      .patch("/api/articles/1")
+      .send(newVote)
+      .expect(200)
+      .then((response) => {
+        expect(response.body).toMatchObject(expectedArticle1);
+        expect(response.body.votes).toBe(100);; // i have put 100 as the test before simulates an increase in articled 1
```

Comment:
Once you have fixed the beforeAll issue you will need to update this test accordingly